contrib/intel/jenkins/summary.py

```diff
@@ -430,6 +431,23 @@ def check_fail(self, line):
             self.fails += 1
             self.failed_tests.append(self.name)
 
+    def check_trace(self, line):
```

Comment:
I don't think this is getting called in the OneCCL Summarizer. You probably need to overload the summarize() and check_line() functions from the superclass just like in the Fabtests Summarizer. This will also move your final check in this function to the summarize() function like in fabtests so the print only happens once.